helm/cloud-provider-vsphere/values.yaml

```diff
@@ -2,105 +2,106 @@
 # This is a YAML-formatted file.
 # vSohere CPI values are grouped by component
 
-global:
+cloud-provider-for-vsphere:
```

Comment:
This PR just moved everything under `cloud-provider-for-vsphere` section. It will be improved as explained in the road map.